Source/animationinfos.h

```diff
@@ -9,6 +9,21 @@
 
 namespace devilution {
 
+/**
+ * @brief Specifies what special logics are applied for a Animation
+ */
+enum class AnimationDistributionParams : uint8_t {
```

Comment:
Any specific reason to have the underlying type be `uint8_t` instead of just `int`?